resources/js/components/modules/media/CardFlip.vue

```diff
@@ -0,0 +1,24 @@
+<template>
```

Comment:
i think it would be better if our templates were in a directory called `templates` instead of media.

So it would look like this:
components
-> modules
   -> templates
       -> quizTemplate.vue
       -> cardFlipTemplate.vue
       -> informativeTemplate.vue